src/st/nz0/e_init.c

```diff
@@ -159,39 +159,39 @@ PfnEntityUpdate OVL_EXPORT(EntityUpdates)[] = {
+u16 g_EInitUnkId13[] = {0x0000, 0x0000, 0x0000, 0x0000, 0x0002};
 u16 D_80180C10[] = {0x0000, 0x0000, 0x0000, 0x0000, 0x0001};
-u16 g_eInitGeneric2[] = {0x0000, 0x0000, 0x0000, 0x0000, 0x0003};
-u16 g_eDamageDisplayInit[] = {0x0000, 0x0000, 0x0000, 0x0000, 0x0003};
+u16 g_EInitCommon[] = {0x0000, 0x0000, 0x0000, 0x0000, 0x0003};
+u16 g_EInitDamageNum[] = {0x0000, 0x0000, 0x0000, 0x0000, 0x0003};
 u16 D_80180C34[] = {0x8001, 0x0000, 0x0000, 0x0000, 0x0003};
-u16 D_80180C40[] = {0x8002, 0x0000, 0x004C, 0x0200, 0x0046};
-u16 g_BoneScimitarInit[] = {0x8003, 0x0001, 0x004D, 0x0204, 0x0069};
-u16 g_ScimitarPartsInit[] = {0x8003, 0x0000, 0x004D, 0x0204, 0x0002};
-u16 g_EAxeKnightInit[] = {0x8004, 0x0001, 0x0048, 0x0207, 0x00F6};
-u16 D_80180C70[] = {0x8004, 0x002F, 0x0048, 0x0205, 0x018F};
-u16 g_EBloodyZombieInit[] = {0x8005, 0x0001, 0x004A, 0x0209, 0x000D};
-u16 D_80180C88[] = {0x8006, 0x0001, 0x004E, 0x020C, 0x004B};
-u16 D_80180C94[] = {0x8006, 0x0000, 0x004E, 0x020C, 0x0002};
-u16 D_80180CA0[] = {0x8006, 0x0015, 0x004E, 0x020C, 0x004C};
-u16 D_80180CAC[] = {0x8007, 0x0034, 0x0050, 0x020D, 0x0051};
-u16 D_80180CB8[] = {0x8007, 0x0040, 0x0050, 0x020D, 0x0052};
-u16 D_80180CC4[] = {0x8008, 0x0000, 0x004B, 0x0219, 0x0005};
-u16 D_80180CD0[] = {0x8009, 0x0000, 0x004F, 0x0211, 0x0005};
-u16 D_80180CDC[] = {0x800A, 0x0000, 0x0053, 0x0215, 0x0005};
-u16 D_80180CE8[] = {0x800C, 0x0000, 0x004C, 0x021D, 0x0005};
-u16 D_80180CF4[] = {0x800C, 0x0000, 0x004C, 0x021D, 0x0002};
-u16 D_80180D00[] = {0x8001, 0x0008, 0x0000, 0x0000, 0x0005};
-u16 D_80180D0C[] = {0x800D, 0x0000, 0x0048, 0x0227, 0x00F3};
-u16 D_80180D18[] = {0x800D, 0x0000, 0x0048, 0x0227, 0x00F4};
-u16 D_80180D24[] = {0x800D, 0x0000, 0x0048, 0x0227, 0x00F5};
-u16 D_80180D30[] = {0x800E, 0x0000, 0x004C, 0x022A, 0x00FE};
-u16 D_80180D3C[] = {0x0000, 0x0000, 0x0000, 0x0000, 0x00FF};
-u16 D_80180D48[] = {0x0000, 0x0000, 0x0000, 0x0000, 0x0100};
+u16 g_EInitBloodySkeleton[] = {0x8002, 0x0000, 0x004C, 0x0200, 0x0046};
+u16 g_EInitBoneScimitar[] = {0x8003, 0x0001, 0x004D, 0x0204, 0x0069};
+u16 g_EInitScimitarParts[] = {0x8003, 0x0000, 0x004D, 0x0204, 0x0002};
+u16 g_EInitAxeKnight[] = {0x8004, 0x0001, 0x0048, 0x0207, 0x00F6};
+u16 g_EInitAxeKnightAxe[] = {0x8004, 0x002F, 0x0048, 0x0205, 0x018F};
+u16 g_EInitBloodyZombie[] = {0x8005, 0x0001, 0x004A, 0x0209, 0x000D};
+u16 g_EInitSkeleton[] = {0x8006, 0x0001, 0x004E, 0x020C, 0x004B};
+u16 g_EInitSkeletonPieces[] = {0x8006, 0x0000, 0x004E, 0x020C, 0x0002};
+u16 g_EInitSkeletonBone[] = {0x8006, 0x0015, 0x004E, 0x020C, 0x004C};
+u16 g_EInitSpittleBone[] = {0x8007, 0x0034, 0x0050, 0x020D, 0x0051};
+u16 g_EInitSpittleBoneSpit[] = {0x8007, 0x0040, 0x0050, 0x020D, 0x0052};
+u16 g_EInitTableWithGlobe[] = {0x8008, 0x0000, 0x004B, 0x0219, 0x0005};
+u16 g_EInitLifeMaxTank[] = {0x8009, 0x0000, 0x004F, 0x0211, 0x0005};
+u16 g_EInitPrizeContainer[] = {0x800A, 0x0000, 0x0053, 0x0215, 0x0005};
+u16 g_EInitSubwpnCloche[] = {0x800C, 0x0000, 0x004C, 0x021D, 0x0005};
```

Comment:
I'd never heard of the term cloche but it seems appropriate. Not sure if there's a more common term in English or not. Image from google image search:
![image](https://github.com/user-attachments/assets/996816c6-2304-4bac-992f-94255f30c667)

Reply:
I only know it as those things from fancy restaurants to hide the food
![image](https://github.com/user-attachments/assets/f49d800f-c75d-4d2c-accf-429eba746eb0)